pyspectrafuse/mgf_convert/parquet2mgf.py

```
@@ -113,28 +113,54 @@ def convert_to_mgf(self, parquet_path: str, sdrf_path: str, output_path: str,
             mgf_group_df['spectrum'] = row_group.apply(
                 lambda row: Parquet2Mgf.get_spectrum(row, ParquetPathHandler(parquet_path).get_item_info()), axis=1)
 
-            # read sdrf file,get dict
-            sample_info_dict = SdrfUtil.get_metadata_dict_from_sdrf(sdrf_path)
+            def _get_mgf_path(row):
+                fname = Parquet2Mgf.get_filename_from_usi(row)
+                info = sample_info_dict.get(fname)
+                if info is None:
+                    logger.warning(f"No metadata for run file: {fname}")
+                    info = ['Unknown', 'Unknown']
+                return '/'.join(info + ['charge' + str(row["charge"]), 'mgf files'])
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)